src/openrct2/object/RideObject.cpp

```diff
@@ -831,7 +831,7 @@ rct_ride_entry_vehicle RideObject::ReadJsonCar(const json_t * jCar)
         { "VEHICLE_ENTRY_FLAG_OVERRIDE_NUM_VERTICAL_FRAMES", VEHICLE_ENTRY_FLAG_OVERRIDE_NUM_VERTICAL_FRAMES },
         { "VEHICLE_ENTRY_FLAG_13", VEHICLE_ENTRY_FLAG_13 },
         { "VEHICLE_ENTRY_FLAG_SPINNING_ADDITIONAL_FRAMES", VEHICLE_ENTRY_FLAG_SPINNING_ADDITIONAL_FRAMES },
-        { "VEHICLE_ENTRY_FLAG_15", VEHICLE_ENTRY_FLAG_15 },
+        { "VEHICLE_ENTRY_FLAG_LIFT", VEHICLE_ENTRY_FLAG_LIFT },
```

Comment:
What does this mean? I assume it's not the same as the lift ride type.

Reply:
It is. It is only set on the lift cabin, mine lift cabin, and blob-from-outer-space lift cabin.

(Spacek also explained this in the issue)

Reply:
Ah, I didn't see the edit. At first I was thinking it was meant for rides that have a steep hill lift or something, like the giga coaster. This flag seems pretty pointless then, if the ride type can be checked. Same goes for the mini golf flag earlier on in the list.

Reply:
I tested the observation tower on a lift ride and it failed to move. I think the flag is necessary for the lift to work.